index.js

```diff
@@ -17,15 +33,21 @@ module.exports = function Rivet(url, options) {
     )
   }
 
-  function fetch({ query, fragments = [], dependencies = [], variables }) {
+  /**
+   *
+   * @param {Object} params
+   * @param {string} params.query
+   * @param {{ fragmentSpec?: FragmentSpec }[]} [params.dependencies]
+   * @param {Record<string, any>} [params.variables]
+   */
+  function fetch({ query, dependencies = [], variables }) {
```

Comment:
@jmfury What do you think of adding a generic here for the return value? We'd default to `any`, but allow you to pass in a type if you wanted to.

```ts
const { allBrandDownloadCategories } = await rivetQuery<{
  allBrandDownloadCategories: Category[];
}>();
```

Reply:
Sounds great to me!